keyboards/redragon/k552/v2/iso/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2023 Stephan Reichholf (https://github.com/sreichholf)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#define MATRIX_UNSELECT_DRIVE_HIGH
```

Comment:
```suggestion
```
this should ideally be moved in `keyboards/redragon/k552/v2/config.h`
the ISO/ANSI boards share the same configuration, minus the LED config and keymap.